### PR TITLE
[MA-400] Generate and return automated testing config for integration tests

### PR DESCRIPTION
**Pull request by adentong**
#changelog [MA-400] Generate and return automated testing config for integration tests

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [x] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# For PR Reviewer 
- [ ] Reviewed unit tests to make sure we are using real components rather than mocks as much as possible?
- [ ] For any major change (observer, new Bolt feature, core Magento interaction) we must add a feature switch, did you verify this?

# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have created or modified unit tests to sufficiently cover my changes.
- [ ] I have added my Jira ticket link and provided a changelog message.

### CODE REVIEW

**Comment by adentong** [comment]
Literally the only file that has logic is Helper/AutomatedTesting.php

File: Helper/AutomatedTesting.php [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+     *
+     * @param string $type
+     *
+     * @return Product|null
+     */
+    protected function getProduct($type)
+    {
+        $this->sortOrder->setField('price')->setDirection('DESC');
+
+        $searchCriteria = $this->searchCriteriaBuilder
```
Comment (vitaliyreznikov) [inline Helper/AutomatedTesting.php:225]: need to find product with inventory (I think in another PR)

**Review by vitaliyreznikov — APPROVED** [review]
LGTM

File: Helper/AutomatedTesting.php (hunk 1 above)
Comment (adentong) [reply Helper/AutomatedTesting.php:224]: added. its a small addition.

**Review by vitaliyreznikov — APPROVED** [review]

**Pull request merged**